src/lib/BindToStores.react.js

```diff
@@ -1,47 +1,121 @@
+import invariant from 'invariant'
 import React from 'react'
-import StorePrototype from './StorePrototype'
 
-// let's say for now, stores is a dictionary of name and object
-export default function bindResources(Component, resourceName) {
-  function extractStores() { return Object.getOwnPropertyNames(stores).map(store => stores[store]) }
-  function capitalize(word) { return word.charAt(0).toUpperCase() + word.slice(1) }
-  function singularize(word) { return (word.slice(-1) === 's' ? word.slice(0, -1) : word) }
+import Container from '../lib/Container'
+import storePrototype from './StorePrototype'
```

Comment:
It's a function rather than a 'class', so I'm done with using PascalCase for `storePrototype`.